### ISSUE TITLE
Installation Issue  on Sonoma 14.3.1

### ISSUE THREAD

**Issue by alexeysur** [Lost-Entrepreneur439/hp-elitebook-840-g3-hackintosh-ventura#22]
Hello!

Can you please help me? I find it impossible to install it.
I have the same configuration as you:

- Intel 6th generation Core i5-6200U 2.3 GHz, up to 2.8 GHz with Intel Turbo Boost Technology, 3 MB cache, 2 cores
- Integrated Intel HD Graphics 520 
- Non touch 35.6 cm (14 in) diagonal LED backlight FHD Slim eDP SVA Anti-glare (1920 x 1080) with camera
- hard drive 2,5" SSD 
- 8192 MB total system memory (8192 MB x 1)
- Intel Wireless-AC 3165 802.11 a/b/g/n/ac (1 x 1) WiFi and Bluetooth 4.0 combo (non-vPro)
- Intel Ethernet Connection I219-V 10/100/1000 Ethernet 

I have formatted the drive as macOS Extended Journaled as well as APFS with GUID partition maps. 
I'm trying to install Sonoma 14.3.1.  However, the installation process is successful before the first reboot. 
Then it tries to boot, showing the  Apple logo, and goes into rebooting many times.

**Comment by Lost-Entrepreneur439** [comment]
this isn't maintained anymore, as my 840 g3 had motherboard failure a few months back. opencore and the kexts are outdated, which will cause installation issues. rebuild the efi from the dortania guide.

**Comment by alexeysur** [comment]
It's a pity. :((
I think many people would be happy to get the opportunity to install Sonoma on the their laptop HP 840 G3 
Thanks for your answer)

**Comment by nyxxxia** [comment]
> It's a pity. :(( I think many people would be happy to get the opportunity to install Sonoma on the their laptop HP 840 G3 Thanks for your answer)

i actually made a functional efi if you wanna try it??

**Comment by randomappleboi** [comment]
> i actually made a functional efi if you wanna try it??
Yes, that would be great! What are your specs and does it work on macOS Sonoma? Thanks for offering!
